deploy/base/05-configmap.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: v1
```

Comment:
Hmm - I wonder if the base manifests could be soft symlinked to avoid mismatched YAML in the future...